appveyor.yml

```diff
@@ -18,6 +18,9 @@ environment:
 build_script:
   - cd ..
   - git clone --depth=1 https://github.com/xbmc/xbmc.git
+  - IF %GENERATOR% == "Visual Studio 14" (cd xbmc\tools\buildsteps\win32) ELSE (cd xbmc\tools\buildsteps\x64)
+  - download-msys2.bat
```

Comment:
`download-msys2.bat sh`

Reply:
fails at the if. it seems to work in my cmd window. confused.

Reply:
ah, my paths miss a windows subdir.